tools/spec-gen-sdk/src/utils/runScript.ts

```diff
@@ -122,7 +121,7 @@ export const runSdkAutoCustomScript = async (
   }
 
   if(logIssues.length > 0) {
-    vsoLogIssue(removeAnsiEscapeCodes(logIssues.join('\n')));
+    context.logIssues.push({command: scriptPath, logIssues: removeAnsiEscapeCodes(logIssues)});
   }
 
```

Comment:
In function `listenOnStream`, line #156 - #160
```
if (context.config.runEnv === 'azureDevOps' && isLineMatch(line)) {
      logIssues.push(line);
    }
context.logger.log(logType, `${prefix} ${line}`, { showInComment: _showInComment, lineResult });
```

Reply:
- Rely on the regex settings in each SDK language repo for the 'errors' filtering. 
- Always log the lines in the context even if they are inserted into `logIssues` array.

Reply:
pls check new change